### ISSUE TITLE
Add more information on History page

### ISSUE THREAD

**Issue by Pohli** [llelectronics/videoPlayer#45]
If I go to History page I often see "cryptic" filenames of recently streamed videos. I can't even tell if the latest video is at the top or bottom of the list. I think it's always at the bottom but focus is set to top I think. This should be changed to latest video at top.
Can you show some more information on the History page like title of the video grabbed from some meta data or at least a timestamp of last watched?

**Comment by llelectronics** [comment]
Yeah the history needs an update. The cryptic filenames is actually a bug. It adds the last stream played first as normal name and then also as url. 
Not sure how to fix that yet to cover all scenarios (like what to do if there is no streamtitle but only streamurl) 

Better sorting and adding timestamps makes sense however. Thanks for that suggestion.

**Comment by llelectronics** [comment]
Partly solved by: https://github.com/llelectronics/videoPlayer/commit/a6c4d18409814cd538f934ef98b0fce00340a088

Timestamp extraction is currently missing.
